src/get-started/java-binding.md

```diff
@@ -492,7 +492,7 @@ For using the library just include the dependency in your `pom.xml`:
 [service]: https://exonum.com/doc/api/java-binding-core/latest/com/exonum/binding/service/Service.html
 [standardserializers]: https://exonum.com/doc/api/java-binding-common/latest/com/exonum/binding/common/serialization/StandardSerializers.html
 [storage-indices]: https://exonum.com/doc/api/java-binding-core/latest/com/exonum/binding/storage/indices/package-summary.html
-[submittransaction]: https://github.com/exonum/exonum-java-binding/blob/v0.3/exonum-java-binding-cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/ApiController.java
+[submittransaction]: https://github.com/exonum/exonum-java-binding/blob/v0.4/exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/ApiController.java
```

Comment:
Would you double-check if this link is used, when rebase onto our 0.4 branch? Our demo doesn't seem to include such functionality as of 0.4.